communities/epfl-hackers/community-assessment.md

```diff
@@ -0,0 +1,87 @@
+- Is your community open to other topics, or are their interests very specific?
+	- It is quite open to suggestion and will accomodate any topic as long as it is tech related, and there is interest in it.
+
+### Goals
+- Why do people get involved in your community?
+	- Its an opportunity to work on cool project where they have the freedom to work on anything they are intersted in, rather than something they are forced to in their classwork. Its also a support structure as there are other people to guide you if you are stuck or to give you advice - this way you aren't doing this on your own.
+- Are they students hoping to get a degree?
+	- Yes
+- Are they trying to ramp-up to get a job?
+	- Yes, but not all of them
+- Are they passionate about developing, regardless of their major?
+	- Yes
+- What activities and goals can you set for your community based on their existing goals?
+	- Goals
+		- Learn something new
+		- Work on a 'side-project' on your own/with friends
+	- Activities
+		- Participate in hackathons
+		- Have occasional meetups to discuss anything (questions, challenges, doubts about a project, a class, an interview)
+		- Have regular workshops to (usually) learn/practice and (occasionally) teach something.
+
+### Location
+- Where does your community currently gather? Is it an online-only group so far, or is there a place where people can meet?
+	- So far we just talk online - we aren't big enough to have "gatherings". But if/when we meet there are lots of public spaces and easily accessible classrooms at EPFL's campus.
+- Is there parking, and if so, is it free?
+	- Yes
+- Is there public transportation available to and from meet-ups?
+	- Yes
+- Is the location accessible to those both on and off campus?
+	- Yes
+- How much does it cost to use the space?
+	- Free
+- How flexible is the space for different types of presentations and meetings?
+	- Can accomodate everything
+- How many people can it fit?
+	- On a regular basis upto 50 people - in more exceptional cases upto 100.
+- Is there comfortable seating for all shapes and sizes?
+	- Yes
+- Is there water or snacks available nearby?
+	- Yes
+
+### Communication
+- How does your community get in touch with one another?
+	- So far just a facebook chat group. But when the group grows, we'll definitely need a better way.
+	- There is a facebook group for the computer science department at EPFL, but it is quite neglected as it is only used to anounce when exam results are out, or to occasionally advertise some event.
+- Is there an email group, a GitHub organization, a Twitter account, a Facebook group, a chat room or channel?
+	- Nope.
+- Could your organization benefit from one or more of those, or another way of communicating?
+	- Yes, certainly! Slack would be ideal as it can host general chatter (like the facebook group HH:EU), and also more precise channels e.g. "Going to MLH prime", "Node.js enthusiasts", etc.
```

Comment:
Slack can be tricky as it requires a VERY engaged group; what channels do your students use everyday that can you can piggy back? This is why Facebook often works better